plugins/kubernetes/src/utils/clusterLinks.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const KindMappings: any = {
+  deployment: 'deployment',
+  ingress: 'ingress',
+  service: 'service',
+  horizontalpodautoscaler: 'deployment',
+};
+
+export function formatClusterLink(
+  dashboardUrl: string,
+  object: any,
+  kind: string,
+) {
```

Comment:
an interface like this is always nicest to make into an object instead, especially since you then more nicely can make `dashboardUrl` actually optional.

```ts
export function formatClusterLink(options: {
  dashboardUrl?: string,
  object: any,
  kind: string,
}) {
```

Reply:
I like it, smart suggestion! Makes reading the unit tests a lot easier too!